lib/proxy/log.rb

```diff
@@ -1,6 +1,13 @@
 require 'logger'
+begin
+  require 'syslog/logger'
+rescue LoadError
+  puts "Setting log_file=SYSLOG not supported on this platform"
+end
 
-Logger.class_eval { alias_method :write, :'<<' } # ::Rack::CommonLogger expects loggers to implement 'write' method
+# ::Rack::CommonLogger expects loggers to implement 'write' method
+Logger.class_eval { alias_method :write, :info }
+::Syslog::Logger.class_eval { alias_method :write, :info }
```

Comment:
This isn't going to work on rubies < 2.0.0: there's no `Syslog::Logger` on these platforms, and what exists wildly differs from the interface used now. That's the reason behind tests breakage (also proxy won't start on older rubies now either).


I think